pages/search/_query/index.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-	<div class="container">
+	<article>
```

Comment:
I have completed this change in `pages/search/_query/index.vue` and `pages/tag/_query/index.vue` already for an upcoming PR for issue #115. I will push it later today